script/tool/lib/src/lint_android_command.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/file.dart';
+import 'package:flutter_plugin_tools/src/common/plugin_utils.dart';
+import 'package:platform/platform.dart';
+
+import 'common/package_looping_command.dart';
+import 'common/process_runner.dart';
+
+const String _gradleWrapper = 'gradlew';
+
+/// Lint the CocoaPod podspecs and run unit tests.
+///
+/// See https://guides.cocoapods.org/terminal/commands.html#pod_lib_lint.
+class LintAndroidCommand extends PackageLoopingCommand {
+  /// Creates an instance of the linter command.
+  LintAndroidCommand(
+    Directory packagesDir, {
+    ProcessRunner processRunner = const ProcessRunner(),
+    Platform platform = const LocalPlatform(),
+  }) : super(packagesDir, processRunner: processRunner, platform: platform);
+
+  @override
+  final String name = 'lint-android';
+
+  @override
+  final String description = 'Runs "gradlew lint" on Android plugins.\n\n'
+      'Requires the example to have been build at least once before running.';
```

Comment:
I think it only requires `flutter pub get`. Could this command invoke it before running ./gradlew?

Reply:
do you remember what file was missing?

Reply:
Yep, `gradlew`.

If you `git clean` a Flutter app (which removes `android/gradlew`), then `flutter pub get` or `flutter packages get` there's still no `android/gradlew`.